Berksfile

```diff
@@ -3,7 +3,12 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'java'
-  cookbook 'minitest-handler'
-  cookbook 'stash_test', :path => 'test/cookbooks/stash_test'
+  cookbook 'java',
+    # Pending release: ark_java fails when tar is not installed
+    # See: https://github.com/agileorbit-cookbooks/java/issues/263
+    github: 'agileorbit-cookbooks/java'
```

Comment:
My only concern of pinning the other cookbook will raise conflict.

e.g. java
Other cookbook may depend to an older version if we pin to the github HEAD. 

And if we pinned to a specific version,  Berkshelf may not handle the dependency resolution well.